test/test_integration.py

```diff
@@ -39,7 +52,24 @@ def teardown_module():
     for test in tests:
         shutil.rmtree(os.path.join(HOME_DIR, "raw_data", test['name']))
         os.remove(os.path.join(HOME_DIR, "scripts", test['name'] + '.script'))
-        os.remove(test['name']+'_'+test['name']+'.txt')
+        os.system("rm -r *{}".format(test['name']))
+
+
+def get_csv_string(dataset, engines, tmpdir, db):
```

Comment:
We should add a docstring here.

What do you think about `get_output_as_csv` instead of `get_csv_string`? The `_output` version is more intuitive to my brain.

Draft of docstring:

```
Install dataset and return the output as a string version of the csv

The string version of the csv output returned by this function can be compared
directly to the expect_out values in the dataset test dictionaries.
```